ui/lib/core/addon/components/external-link.js

```diff
@@ -0,0 +1,24 @@
+import Component from '@glimmer/component';
+
+/**
+ * @module ExternalLink
+ * `ExternalLink` components are used to render anchor links to non-cluster links. Automatically opens in a new tab with noopener noreferrer.
+ * To link to vaultproject.io, use DocLink. To link to learn.hashicorp.com, use LearnLink.
+ *
+ * @example
+ * ```js
+    <ExternalLink @href="https://hashicorp.com">Arbitrary Link</ExternalLink>
+ * ```
+ *
+ * @param href="https://example.com/"{String} - The full href with protocol
+ * @param sameTab=false {Boolean} - by default, these links open in new tab. To override, pass @sameTab={{true}}
+ *
+ */
+export default class DocLinkComponent extends Component {
```

Comment:
Probably just haven't gotten to yet, but I think this should be 
```suggestion
export default class ExternalLinkComponent extends Component {
```